src/SFA.DAS.Admin.Aan.Web/Infrastructure/RouteNames.cs

```diff
@@ -14,5 +14,8 @@ public static class CreateEvent
         public const string EventType = nameof(EventType);
         public const string EventDescription = nameof(EventDescription);
         public const string EventHasGuestSpeakers = nameof(EventHasGuestSpeakers);
+        public const string GuestSpeakerList = nameof(GuestSpeakerList);
+        public const string GuestSpeakerAdd = nameof(GuestSpeakerAdd);
+        public const string GuestSpeakerDelete = nameof(GuestSpeakerDelete);
```

Comment:
You can change this now or later, but we will have similar route for edit event, so this will require to have `CreateEvent` suffixes.

Reply:
let's do it later